README.md

```diff
@@ -16,3 +16,4 @@ Main purpose of this repository is collaboration of homework for study groups
 ## CI/CD
 
 In preparing
+# 1
```

Comment:
Do not do changes in main readme file

Reply:
Please remove changes in this readme file

Reply:
fixed